CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.2.15
+
+## Fixes
+
+* Fixes a bug where not all data points would be read if a time series had infrequent data points.
```

Comment:
Maybe we should be more specific about how infrequent they had to be for this bug to occur?